modules/loader/src/main/java/com/opengamma/strata/loader/csv/FullSwapTradeCsvLoader.java

```diff
@@ -660,6 +663,23 @@ private static PriceIndexCalculationMethod parseInflationMethod(Optional<String>
         .map(cal -> BusinessDayAdjustment.of(dateCnv, cal));
   }
 
+  // days adjustment, defaulting business day convention and holiday calendar
+  private static BusinessDayAdjustment parseBusinessDayAdjustmentWithFallback(
+      CsvRow row,
+      String leg,
+      String cnvField,
+      String calField) {
+
+    Currency currency = Currency.of(getValueWithFallback(row, leg, CURRENCY_FIELD));
+    BusinessDayConvention dateCnv = findValue(row, leg, cnvField)
+        .map(s -> LoaderUtils.parseBusinessDayConvention(s))
+        .orElse(BusinessDayConventions.MODIFIED_FOLLOWING);
+    HolidayCalendarId holidayCalendarId = findValue(row, leg, calField)
+        .map(s -> HolidayCalendarId.of(s))
+        .orElseGet(() -> HolidayCalendarId.defaultByCurrency(currency));
```

Comment:
These defaults are not documented anywhere AFAIK. eg. [here](http://strata.opengamma.io/holiday_data/) or [here](http://strata.opengamma.io/trade_loader/)

Reply:
How does one modify the documentation?

Reply:
https://github.com/OpenGamma/StrataDocs